src/console/CmdProcessor.cpp

```diff
@@ -264,9 +285,17 @@ void CmdProcessor::printHeader(
 
     size_t idx = 0;
     for (auto& cname : (*resp.get_column_names())) {
-        std::string fmt = folly::stringPrintf(" %%-%lds |", widths[idx++]);
+        auto len = widths[idx++];
+        if (curSpaceCharset_.compare("utf8") == 0) {
+            auto byteLen = cname.size();
+            auto charLen = CharsetInfo::getUtf8Charlength(cname);
+            auto diff = (byteLen - charLen) / 2;
+            len += diff;
```

Comment:
Why we add diff here but minus diff in `calColumnWidths`?

Reply:
One less byte to display when displaying a Chinese character